src/bundles/general.stanza

```diff
@@ -152,3 +152,31 @@ Reset Interface Bundle
 <DOC>
 public pcb-bundle reset :
   pin reset
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; Device Classes
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+doc: \<DOC>
+Polarized Bundle Type
+
+This bundle attempts to mimic the expected
+interface for other polarized devices like
+capacitors, diodes etc by presenting the `c/a`
+ports.
+
+This bundle is useful when defining ports on
+components like a RJ-45 connector that has
+status LEDs embedded into it. This can be
+used to define a port that can represent
+the polarized nature of the device.
+
+Also works for supports/require pin assignment
+for arrays of equivalent devices.
+
+@member c Cathode (negative) port
+@member a Anode (positive) port
+<DOC>
+public pcb-bundle polarized-dev :
```

Comment:
or `polarized-comp` for component

Reply:
I tried just `polarized` at first but it seemed to conflict the `polarized` method that I had defined in `transformer.stanza` for some reason. Stanza seemed to think this bundle definition was ambiguous with that definition.

I don't think `polarized-ports` is a good idea because this is a bundle definition and I don't want to have the user confuse it with a `port` which is already a separate concept in our product.

Reply:
I think the typical approach we have been doing is `polarized-b` with the `-b` suffix implying bundle. That works around some of the symbol ambiguity issues.

I just wasn't feeling `polarized-b` as a name.

Reply:
Maybe `polarized-ca` indicating `c / a` as the cathode/anode.

Reply:
Wait you're a genius that's the answer. I like `polarized-ca`

Also. 

'ca' == your initials. It all makes sense now.